app/routes/apply/new-eligibility/about-the-prisoner.js

```diff
@@ -32,11 +52,12 @@ module.exports = function (router) {
         req.body['dob-year'],
         req.body['PrisonerNumber'],
         req.body['NameOfPrison'])
-
-      insertNewEligibilityAndPrisoner(aboutThePrisoner, req.params.claimType, req.query.reference)
+      insertNewEligibilityAndPrisoner(aboutThePrisoner, claimType, req.query.reference)
         .then(function (result) {
           var referenceId = referenceIdHelper.getReferenceId(result.reference, result.eligibilityId)
-          return res.redirect(`/apply/${req.params.claimType}/new-eligibility/${dob}/${relationship}/${benefit}/${referenceId}`)
+          req.session.referenceId = referenceId
+
+          return res.redirect(`/apply/${claimType}/new-eligibility/${dobEncoded}/${relationship}/${benefit}/${referenceId}`)
```

Comment:
should be using the session to store these params now, so the redirect and about-you route should be like:
`/apply/${claimType}/new-eligibility`